content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/embeddable/youtube/v2/v2.html

```diff
@@ -0,0 +1,29 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Please see remark above: the HTL file should then be at:
`.../embed/embeddable/youtube/v2/youtube/youtube.html`